python/lsst/pipe/base/_instrument.py

```diff
@@ -75,6 +77,13 @@ class Instrument(metaclass=ABCMeta):
     """Instrument specific name to use when locating a policy or configuration
     file in the file system."""
 
+    translatorClass: MetadataTranslator | None = None
```

Comment:
@gpdf are SPHEREx using MetadataTranslator? In the review we were wondering if this should be moved to `obs_base` instead since it might be assuming too much about how people generate exposure records.

Reply:
Not at present.  What is the "this" that you want to move?

Reply:
I have moved all the changes to lsst/obs_base#477